In [ ]:
# 문제 풀이

In [2]:
import pandas as pd
import seaborn as sns

In [4]:
df = sns.load_dataset('titanic')
sample = df.copy()
sample.head()


survived  pclass     sex   age  sibsp  parch     fare embarked  class  \
0         0       3    male  22.0      1      0   7.2500        S  Third   
1         1       1  female  38.0      1      0  71.2833        C  First   
2         1       3  female  26.0      0      0   7.9250        S  Third   
3         1       1  female  35.0      1      0  53.1000        S  First   
4         0       3    male  35.0      0      0   8.0500        S  Third   

     who  adult_male deck  embark_town alive  alone  
0    man        True  NaN  Southampton    no  False  
1  woman       False    C    Cherbourg   yes  False  
2  woman       False  NaN  Southampton   yes   True  
3  woman       False    C  Southampton   yes  False  
4    man        True  NaN  Southampton    no   True

In [6]:
sample['class'].value_counts()

class
Third     491
First     216
Second    184
Name: count, dtype: int64

sample 데이터프레임에 대하여 apply()를 활용하여 class 컬럼의 값을 다음과 같이 바꾸고, 
분포를 출력후 변경 전과 동일한지 확인하세요

삼등석    491
일등석    216
이등석    184
Name: class, dtype: int64